sdk/eventhubs/azure-messaging-eventhubs/samples/README.md

```diff
@@ -0,0 +1,35 @@
+# Eventhub Samples
+
+This repository contains samples for the Azure Event Hubs service.
+
+## Sample Requirements
+
+These samples are written with the assumption that the following environment
+variables have been set by the user:
+
+* EVENTHUBS_CONNECTION_STRING - The service connection string for the eventhubs instance.
+* EVENTHUB_NAME - Name of the eventhubs instance to communicate with.
+* EVENTHUB_HOST - Fully qualified domain name for the eventhubs instance.
```

Comment:
Should these be EVENTHUB`S`_* for consistency?

Reply:
I believe that an eventhubs instance is called an eventhub.  However I think you're right about the host.